src/sass/pages/_search.scss

```diff
@@ -0,0 +1,62 @@
+@use "sass:map";
+@use "../abstracts" as abstracts;
+
+.filter-wrapper {
+  background-color: abstracts.$primary-grey;
+  min-height: 100%;
+  width: 300px;
+
+  .filter-header {
+    background-color: abstracts.$primary-darkgrey;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    width: 100%;
+    padding: 5px 10px;
+    cursor: pointer;
+  }
+
+  .filter-chevron {
+    justify-self: end;
+    align-self: center;
+  }
+
+  .filter-content {
```

Comment:
Tähän vois laittaa vielä saman border-bottomin kun filter-headerissa, sitten on ulkoasultaan oikein jees 🎉

Reply:
Tai itseasiassa vois tehdä classin `.filter-content:not(:last-child)` ja sillä se border-bottom, niin se ei tuu viimeselle lapselle.